addons/im_livechat/views/rating_views.xml

```diff
@@ -11,7 +11,6 @@
                 <filter string="Code" name="resource" context="{'group_by':'res_name'}"/>
             </xpath>
             <xpath expr="//filter[@name='resource']" position="after">
-                <field name="parent_res_name"/>
```

Comment:
Do you have a screenshot of the issue when this is not removed?

Reply:
Yes..Have a look - https://www.awesomescreenshot.com/image/7025723?key=acdf88d8f8b6ced48d546873d386c72f